tests/test_run.py

```diff
@@ -197,3 +202,46 @@ def test_GaAs(check_run,system_GaAs_W90, compare_any_result,calculators_GaAs,Efe
                parameters_K = {'_FF_antisym':True,'_CCab_antisym':True } ,
                   extra_precision = {"berry_dipole_fsurf":1e-6} )   # This is a low precision for the nonabelian thing, not sure if it does not indicate a problem, or is a gauge-dependent thing
 
+
+
+
+@pytest.fixture(scope="session")
+def calculators_Chiral(Efermi_Chiral):
+    calculators  = {}
+    calculators['conductivity_ohmic'] = calc.static.Ohmic(Efermi=Efermi_Chiral)
+    calculators['berry_dipole']       = calc.static.BerryDipole_FermiSea(Efermi=Efermi_Chiral,kwargs_formula={"external_terms":False} )
+    calculators['ahc']       = calc.static.AHC(Efermi=Efermi_Chiral,kwargs_formula={"external_terms":False} )
+    return calculators
+
+def test_Chiral_left(check_run,system_Chiral_left, compare_any_result,Efermi_Chiral,compare_fermisurfer,calculators_Chiral):
+
+    grid_param={'NK':[10,10,4], 'NKFFT':[5,5,2]} 
+    check_run(system_Chiral_left , calculators_Chiral , fout_name="berry_Chiral" , suffix="left-run" ,
+                grid_param = grid_param,
+               parameters_K = {'_FF_antisym':True,'_CCab_antisym':True } , use_symmetry = True,
+            extra_precision = {"Morb":-1e-6},)
+
+
+def test_Chiral_leftTR(check_run,system_Chiral_left_TR, compare_any_result,Efermi_Chiral,compare_fermisurfer,calculators_Chiral):
+    "check that for time-revrsed model the ohmic conductivity is the same, but the AHC is opposite"
+    grid_param={'NK':[10,10,4], 'NKFFT':[5,5,2]} 
+    check_run(system_Chiral_left_TR ,calculators_Chiral , fout_name="berry_Chiral" , suffix="left-run" ,
```

Comment:
If these models do not take much time to run, what about just running left and left_TR calculation and comparing the results directly? Adding `flip_sign` option seems to make `check_run` adds complication to an already complicated function.

Reply:
Not sure, what you mean? t

1) to create another set of reference files for TR case?   

Or 

2) just run here both calculations, and compare them without reading reference files. i.e. comparing the result returned by `run()`, without accessing the files ?

Reply:
I meant 2.